overhauled the docs to remove dead links

nochinator opened the pull request:
Changes proposed in this request:
- fixed most (may have missed a few as there were a lot) of broken links
- updated pages for the weapons
- fixed some grammar or spelling errors
- wrote pages for every game mode
- connected most (may have missed a few) pages together in a more fluid way
- incorporated many unused, or hard-to-find pages that had lots of useful info into the flow of the website
- organized file system into folders, gameplay, editing, server management, and variables
- added variables from https://github.com/redeclipse/base/blob/master/config/usage.cfg into this, although it is not a complete list it's much better than nothing. This will be sorted into better categories and polished up in the future.

IceflowRE commented on variables/Variables-home.md:
```diff
@@ -0,0 +1,1122 @@
+Variables can be used to change almost everything about how the game works. Currently there is almost no documentation on the website, so as of now this page is just a really long list of variables and descriptions copied and pasted from [here](https://github.com/redeclipse/base/blob/master/config/usage.cfg) and then formatted for the website. However, we are working on sorting these into a serie ...
```
I would also leave it at the link, the link, as the this list will be outdated pretty fast.
Probably explain some commands here in more in depth as it is in the usage.cfg.

nochinator replied:
ok, I can change the link to be the URL, however, I hardly know any commands/variables so that's why this is just a copy and paste. something is better than nothing

qreeves commented on .vscode/settings.json:
```diff
@@ -0,0 +1,18 @@
+        "unconnectable",
+        "Unported"
+    ]
+}
```
I'm going to have to ask you to remove this file from the pull request as `.vscode` shouldn't be included in the repository. I will have to add a `.gitignore` entry for it.

qreeves commented on FAQ.md:
```diff
@@ -124,9 +124,13 @@ It is easy to hit someone with a shotgun, but the weapon is really only effectiv
+## Why are there no health pickups?
 
-In Red Eclipse, players have an unlimited supply of weapon reloads, health regeneration and [impulse](Impulse-Guide.md) energy. Therefore, map control is less important, and everyone can focus on the real fun: Capturing that flag, grabbing that bomber ball… or just fragging!
+In Red Eclipse, health regeneration and [impulse](Parkour-Guide.md) energy. Therefore, map control is less important, and everyone can focus on the real fun: Capturing that flag, grabbing that bomber ball… or just fragging!
```
The first sentence here doesn't make much sense.

qreeves commented on gameplay/Capture-the-Flag.md:
```diff
@@ -9,6 +9,7 @@
 -   A buff is also applied when carrying either your own flag, or the enemy flag.
 -   You can retrieve your own flag from the spawn area by pressing **F**, making capture for enemies more difficult.
 -   Press **F** to throw away any flags in your possession, or to pass the flag to teammates.
+-   You cannot capture the opposing teams flag when your teams flag is not at any base.
```
Words suffixed with 's' to signify ownership requires an apostrophe `s/teams/team's/`.

qreeves commented on gameplay/Gameplay-Guide.md:
```diff
@@ -1,4 +1,4 @@
-Red Eclipse features 5 playable game modes which can be further customised through mutators. Along with the cooperative map editor and a demo viewer to playback previously recorded matches.
+Red Eclipse features 5 playable game modes which can be further customized through mutators. Along with the cooperative map editor and a demo viewer to playback previously recorded matches.
```
We use 's' instead of 'z' in most UK spelling cases, but if you've changed them all don't worry about changing it back.

qreeves commented on gameplay/Race.md:
```diff
@@ -0,0 +1,17 @@
+
+### Tips & strategy
+
+-   Attempt to complete the course as quickly as possible, however don't rush and get killed by at trap as there is a somewhat large respawn timer.
```
Did you mean `a` instead of `at`?

qreeves commented on gameplay/Status-Effects.md:
```diff
@@ -23,7 +23,7 @@ Some weapons induce status ailments that inflict some damage over time and last
 <img src="images/icons/shock.png" width="48px"/>
 
-Slows down movement greatly. Deals 2 damage per second for 5.5 seconds. Caused by the <img src="images/weapons/zapper.png" width="32px"/>zapper, <img src="images/weapons/mine.png" width="32px"/>mines and specific hurt residuals.
+Slows down movement greatly. Deals 3 damage per second for 5.5 seconds. Caused by the <img src="images/weapons/zapper.png" width="32px"/>the zapper's secondary, <img src="images/weapons/mine.png" width="32px"/>mines and specific hurt residuals.
```
Double `the`.

qreeves commented on gameplay/Status-Effects.md:
```diff
@@ -33,4 +33,4 @@ Deals 3 damage per second for 5.5 seconds. Can be negated with the secondary fir
 
-Deals 3 damage per second for 5.5 seconds. Caused by the secondary fire of the <img src="images/weapons/shotgun.png" width="32px"/>shotgun, <img src="images/weapons/sword.png" width="32px"/>sword and specific hurt residuals. 
\ No newline at end of file
+Deals 3 damage per second for 5.5 seconds. Caused by the secondary fire of the <img src="images/weapons/shotgun.png" width="32px"/>the shotgun's secondary, <img src="images/weapons/sword.png" width="32px"/>the sword's secondary and specific hurt residuals. 
```
Double `the` again.

qreeves commented on gameplay/Weapons-Guide.md:
`s/deal/deals/`

qreeves commented on sever-how-to/Server-Setup.md:
```diff
@@ -70,7 +70,7 @@ For the installation and configuration steps follow the specific guides.
 
 ## Configuration Files
-The configuration files are written in [cubescript](.md), lines starting with `//` are ignored. If you want to configure a setting simply remove the `//` in front of it and change it's value.
+The configuration files are written in cubescript, lines starting with `//` are ignored. If you want to configure a setting simply remove the `//` in front of it and change it's value.
```
`s/it's/its/`

nochinator replied:
there is not "cubescript.md" file, and I don't know cubescript so I can't write that article

qreeves reviewed (changes requested):
Other than the review comments attached, the only other thing I'd suggest is that if you're going to move the editing documents into their own folder, they don't really need the `Editing-` prefix on the filenames themselves anymore as it's a duplication.

qreeves commented:
You'll need to revert `.gitignore` before I can merge as I've already changed it upstream to include more matches.

![image](https://user-images.githubusercontent.com/3297929/216330980-c84052cc-5b96-48d7-a53b-282ca7a1449a.png)

Otherwise, looks good to me, ready for merge once resolved.

nochinator commented:
I've fixed the problem and this should be ready to merge. I can't figure out how to get the "1 change requested" to go away, but I fixed what it says the change is.

qreeves commented on .gitignore:
```diff
@@ -18,13 +18,3 @@ out
 *.orig
 *.rej
 *.bak
-*.kate-swp
```
This is reverting changes I made upstream to ignore the `.vscode` directory and other unwanted files.

nochinator replied:
oh, well ok then

qreeves commented on .vscode/settings.json:
```diff
@@ -0,0 +1,18 @@
+        "unconnectable",
+        "Unported"
+    ]
+}
```
This file is still present in the latest revision.

nochinator replied:
I have removed the file

qreeves commented:
All good now, thanks for your contribution and cooperation.
